daffodil-cli/src/it/resources/org/apache/daffodil/CLI/infoTest.tdml

```diff
@@ -22,6 +22,7 @@
   xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" xmlns:xs="http://www.w3.org/2001/XMLSchema">
 
   <tdml:defineSchema name="s1">
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
```

Comment:
FYI: Every single test used to get this include invisibly (the TDML Runner did that, now it doesn't).

Now all tests must explicitly include things.